### DISCUSSION TITLE
Z-AI

### DISCUSSION THREAD

**Discussion by ghost** [TheIllusiveC4/Curios#396] [General]
Hello, I asked the neural network for the most hardcore zombie mod. He pointed to Z-AI, but it's not online. All links provided by the neural network are invalid. He indicated that you are the author of this mod. Where did this mod go and can I still download it? With respect Taksilia

**Comment by TheIllusiveC4** [comment]
I have no idea what this is referring to, but I've never created a mod by this name.
